addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductScreen.js

```diff
@@ -161,13 +154,14 @@ odoo.define('point_of_sale.ProductScreen', function(require) {
             const { mode } = event.detail;
             NumberBuffer.capture();
             NumberBuffer.reset();
-            this.state.numpadMode = mode;
+            this.env.pos.PRODUCT_SCREEN.numpadMode = mode;
```

Comment:
It doesn't seems to have other numpadMode for any other screen, so maybe you can store it in "this.env.pos.numpadMode"

Reply:
I can do that.